docker/Dockerfile.olsolr

```diff
@@ -1,13 +1,18 @@
 FROM ubuntu:xenial
 
 RUN apt-get -qq update && \
-    DEBIAN_FRONTEND=noninteractive apt-get install -y solr-tomcat haproxy && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y solr-tomcat haproxy cron && \
     ln -s /var/log/tomcat7/ /usr/share/tomcat7/logs && \
     ln -s /etc/tomcat7/ /usr/share/tomcat7/conf
 
 COPY conf/solr/conf/* /etc/solr/conf/
 COPY conf/solr/haproxy.cfg /etc/haproxy/
 COPY conf/solr/start.sh /start.sh
 
+# Log management:
+COPY conf/solr/rmlog /etc/cron.daily/rmlog
+RUN chmod 755 /etc/cron.daily/rmlog
```

Comment:
We shouldn't need to make this executable from here; try running the chmod command outside of the docker image: `chmod +x conf/solr/rmlog.sh`

Reply:
That's good to know, and it worked perfectly.